samples/cellular/modem_shell/src/sock/sock.c

```diff
@@ -354,7 +354,8 @@ static int sock_set_tls_options(
 	uint32_t sec_tag,
```

Comment:
This is not related to your PR, but while you're anyway making changes, could you remove including `nrf_socket.h` from this file? It shouldn't be needed for anything and removing it would prevent accidentally using for example incorrect defines. Everything is not aligned between the POSIX socket API and nRF socket API.